### DISCUSSION TITLE
arm64 Architecture support

### DISCUSSION THREAD

**Discussion by fabiom91** [tiangolo/meinheld-gunicorn-docker#133] [Questions] [answered]
Hi there, M1 macs and raspberry pi use an `arm64` architecture instead of an `amd64` one. This makes it very difficult if not impossible to run your image. Is it possible to add the possibility of specifying the architecture of the image? Or can you add a new docker image file such as: `FROM --platform=linux/arm64 python:3.9`?

**Comment by hemna** [comment]
It would be nice if the docker hub builds had the arm64 architecture as well.

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile: 

There's been support for Arm for a while, but I didn't come here to comment then. :nerd_face:
